server/chat-plugins/rock-paper-scissors.tsx

```diff
@@ -61,52 +67,72 @@ export class RPSGame extends Rooms.RoomGame {
 	}
 	sendControls(player: RPSPlayer) {
 		if (!this.roundTimer) {
-			player.sendRoom(
-				`|controlshtml|<center>The game is paused.<br /><br />` +
-				`<button class="button" name="send" value="/rps resume">Resume game</button></center>`
-			);
-			return;
+			return player.sendControls(<div style={{textAlign: 'center'}}>The game is paused.<br /><br />
+				<button className="button" name="send" value="/rps resume">Resume game</button>
+				</div>);
 		}
 		if (player.choice) {
-			player.sendRoom(`|controlshtml|<center>You have selected <strong>${player.choice}</strong>. Now to wait for your foe.</center>`);
+			player.sendControls(
+				<div style={{textAlign: 'center'}}>You have selected <strong>{player.choice}</strong>. Now to wait for your foe.</div>
+			);
 			return;
 		}
-		let buf = `|controlshtml|<center><strong>Make your choice, quick! You have ${Chat.toDurationString(TIMEOUT)}!</strong><br />`;
-		for (const choice of ['Rock', 'Paper', 'Scissors']) {
-			buf += `<button class="button" name="send" value="/choose ${choice}" style="width:6em"><span style="font-size:24px">${ICONS[choice]}</span><br />${choice || '&nbsp;'}</button> `;
-		}
-		buf += `<br /><br /><button class="button" name="send" value="/rps end">End game</button></center>`;
-		player.sendRoom(buf);
+		player.sendControls(
+			<div style={{textAlign: 'center'}}><strong>Make your choice, quick! You have {Chat.toDurationString(TIMEOUT)}!</strong><br />
+			{['Rock', 'Paper', 'Scissors'].map(choice => (
+				<button className="button" name="send" value="/choose ${choice}" style={{width: '6em'}}>
+				<span style={{fontSize: '24px'}}>{ICONS[choice]}</span><br />{choice || ' '}</button>
+			))}<br /><br /><button className="button" name="send" value="/rps end">End game</button></div>
+		);
 	}
 	getField() {
 		if (this.players.length < 2) {
-			return `|fieldhtml|<center><h2>Waiting to start the game...</h2></center>`;
+			return <div style={{textAlign: 'center'}}><h2>Waiting to start the game...</h2></div>;
 		}
 
 		const [p1, p2] = this.players;
 
 		function renderBigChoice(choice: string, isWinner?: boolean) {
-			return `<div style="width:180px;font-size:120px;background:${isWinner ? '#595' : '#888'};color:white;border-radius:20px;padding-bottom:5px;margin:0 auto">${ICONS[choice] || '&nbsp;'}<br /><small style="font-size:40px"> <small style="font-size:32px;display:block">${choice || '&nbsp;'}</small></div>`;
+			return <div style={{
+				width: '180px',
+				fontSize: '120px',
+				background: isWinner ? '#595' : '#888',
+				color: 'white',
+				borderRadius: '20px',
+				paddingBottom: '5px',
+				margin: '0 auto',
+			}}>{ICONS[choice] || ' '}<br />
```

Comment:
Is that just a space? I don't think a space would do it.

```suggestion
			}}>{ICONS[choice] || '\u00A0'}<br />
```